packages/core/src/sass/app/modules/page-404.scss

```diff
@@ -2,5 +2,8 @@
 .page-404 {
     &__image {
         margin-right: 1.6rem;
+        @media screen and (max-width: 758px) {
+            margin: 0;
+        }
```

Comment:
do you still need these now?

Reply:
Yes, because of image center on small screens, I get rid of margin